config/routes.rb

```diff
@@ -5,59 +5,8 @@
 
   devise_for :users
   resources :metrics
+  resources :features
```

Comment:
Consider restricting the feature resource routes to only those that have actions in the `FeaturesController` like how you've done with the user resource routes on line 9.

(By keeping the app free of unused routes, there is some small benefit in reducing the likelihood of creating broken links and reducing the cognitive load when reviewing `rake routes` output.)